### ISSUE TITLE
[BUG] When I try to run `python src/main.py` 

### ISSUE THREAD

**Issue by Virnnar** [da-rth/yasb#120]
When I type python src/main.py
![image](https://github.com/da-rth/yasb/assets/109610268/de6e0f08-e424-4851-8943-4c9763c72ff4)

**Comment by theSoberSobber** [comment]
yep, it's a version problem, python is an extremely fragile language and breaking changes from dependencies is extremely common, using an older commit would be the best you can probably do.

**Comment by theSoberSobber** [comment]
Update: Reinstalling PyQt6 seems to make it work again, I really really hate python, anyways just run this `pip install PyQT6 --force`.

**Comment by zhulaee** [comment]
thanks， install cmd solved my problem

**Comment by Rakyarvik** [comment]
On my computer, this is an issue caused by the different versions of PyQt6 and PyQt6 Qt6. The requirements. txt file specifies that the version of pyqt6 is 6.3.1. When we enter the pip list, we will see that the versions of PyQt6 and PyQt6 Qt6 are different. Please reinstall version 6.3.1 of PyQt6 Qt6.`pip uninstall PyQt6-Qt6`,`pip install pyqt6-qt6==6.3.1`
